client/batch_test.go

```diff
@@ -1662,8 +1663,7 @@ func mutexClearRegression(t *testing.T, c *test.Cluster, client *Client) {
 		t.Fatalf("getting batch: %v", err)
 	}
 
-	col := uint64(0)
-	row := uint64(1)
+	var col, row uint64
```

Comment:
perhaps the value didn't matter, but it's worth checking that `row` doesn't need to be `1`

Reply:
in both cases row is immediately assigned a new value before being used